### DISCUSSION TITLE
Mars!!!

### DISCUSSION THREAD

**Discussion by lawrencegripper** [curl/curl#6924] [General]
Listened to the recent Changelog podcast where you talked about wanting to know if CURL was used in space ... well looks like it is congrats!

https://github.blog/2021-04-19-open-source-goes-to-mars/

**Comment by bagder** [comment]
Yeps. Some of us now have this badge on github:

![Mars-2020-helicopter-contributor](https://user-images.githubusercontent.com/177011/115348613-1e082400-a1b3-11eb-9d8a-aad77b8fa9d6.png)

It's not clear exactly who gets it and who doesn't... :thinking: I [blogged about mars](https://daniel.haxx.se/blog/2021/04/19/mars-2020-helicopter-contributor/) too.
